lib/get_more_page_data.tsx

```diff
@@ -0,0 +1,31 @@
+import Router from 'next/router'
+
+export default async function getMorePageData(getDataParams) {
+  //getDataParams contains:
+  //dataLength -> needed to skip the data we already loaded before
+  //setData -> set the new data
+  //setDataLength -> set the new data length
+  //setHasMore -> set it "false" if we have no longer data
+  //setStatuses -> set the statuses ids
+  //resultSource -> check data type
+
+  fetch('/api/utils/get_paginated_data?skip=' + getDataParams.dataLength + 
+    '&resultSource=' + getDataParams.resultSource, {
+    method: 'GET'
+  })
+  .then(response => response.json())
+  .then((response) => {
+    if (response.error === "No access!") {
+      Router.push('/')
+    } else {
+      if (response.data.length < 10) {
+        getDataParams.setHasMore(false)
+      }
+      getDataParams.setDataLength(getDataParams.dataLength + 10)
+      getDataParams.setData((bookings) => [...bookings, ...response.data])
+      if (getDataParams.resultSource === "bookings") {
```

Comment:
why are we getting statusTypes here?

Reply:
you told me to get the status when we get bookings. Should I change this?